### PR TITLE
CI: Add WASM target to Clippy job

### LINKED ISSUE #145: Further updates to CI
### Description

Discussions gathered around:

-  ~~CI check for debug features to the matrix (144)~~
- run the clippy check as a matrix on different platforms to capture differences in the build path on different platforms (since Bevy handles the abstraction over desktop platforms, we'd be good just checking Linux and web)

### Alternatives & Prior Art

_No response_

### PR DESCRIPTION
**Pull request by 64kramsystem**
Closes #145.

### CODE REVIEW

**Comment by 64kramsystem** [comment]
Need to install the wasm target :roll_eyes:

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -30,6 +36,13 @@ jobs:
               libasound2-dev \
               libudev-dev
       - uses: actions/checkout@v3
+      - name: Use WASM target for Clippy
```
Comment (zicklag) [inline .github/workflows/ci.yml:39]:
```suggestion
      - name: Install WASM Target
```

Small nit. I feel like this makes the purpose of the stage more clear.

**Review by zicklag — CHANGES_REQUESTED** [review]
Teeny tiny change, but otherwise looks good!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (64kramsystem) [reply .github/workflows/ci.yml]: Updated :+1:

**Review by zicklag — APPROVED** [review]

**Comment by zicklag** [comment]
bors retry

**Pull request merged**